src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/OperatorPrecedence.cs

```diff
@@ -27,6 +27,7 @@ internal enum OperatorPrecedence
         Multiplicative,
         Switch,
         Range,
+        Slice,
```

Comment:
:memo: I don't know what is the correct precedence order in this list.

Reply:
it feels like this shoudl be higher up right?  i.e. if i have `.. y or z` then it's parsed a `.. (y or z)`  not `(..y) or z`.  So `..` has lower precedence than than all the binaries.  so i would expect this to at least be above ConditionalOr.  We should probably note that `..` as a range and `..` as a slice are very different wrt precedence.